src/styles/card.scss

```diff
@@ -0,0 +1,123 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  place-content: center;
+  padding: 50px 40px;
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+
+    &__card:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &__card:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+
+  .card {
+    // width: 200px;
+    box-sizing: border-box;
+    border: 1px solid $border-color;
+    background-color: $bg-primary-color;
+    border-radius: 5px;
+    padding: 32px 16px 16px;
+    transition: all 300ms;
```

Comment:
It is better not to set transitions for everything, it affects performance. It is better to be specific.